Localization Manager

Swan opened the pull request:
Adds a class to manage localization and also unit tests

YaLTeR commented:
Do you think it really is a good idea to roll out our own solution? I suspect we'll bump into an issue with some context-dependent string really quickly.

Swan commented:
Did a bit more testing with other languages, and yes there were some issues with parsing. Replaced it with [ini-parser](https://github.com/rickyah/ini-parser) and added a few more tests for languages like Japanese and Arabic, so it should be fine now.

YaLTeR commented:
Oh, no, I don't mean parsing. I mean things like "You have {i} mapsets available!" where English has two versions (for "1 mapset" and "x mapset**s**") and Russian has three versions, or "You can't put this {objectName} here" where English has one version, but Russian might have two or three versions depending on the gender of `objectName`.
